stdlib/public/SwiftShims/System.h

```diff
@@ -102,22 +102,33 @@
 // Only the bottom 56 bits are used, and heap objects are eight-byte-aligned.
 #define SWIFT_ABI_X86_64_SWIFT_SPARE_BITS_MASK 0xFF00000000000007ULL
 
-// Objective-C reserves the high and low bits for tagged pointers.
-// Systems exist which use either bit.
-#define SWIFT_ABI_X86_64_OBJC_RESERVED_BITS_MASK 0x8000000000000001ULL
+// Objective-C normally reserves the low bit for tagged pointers, but
+// instead reserves the high bit on iOS simulators.
```

Comment:
I don't think this is correct; doesn't it use the high bit everywhere but x86_64 Mac, so that objc_msgSend can do the clever "branch if nil or negative" trick?